commons/src/main/java/com/powsybl/commons/config/PlatformConfig.java

```diff
@@ -117,7 +124,13 @@ public static ModuleConfigRepository loadDefaultModuleRepository() {
     }
 
     public static synchronized PlatformConfig defaultConfig() {
+        ServiceLoader<DefaultConfiguration> defaultConfigProvider = ServiceLoader.load(DefaultConfiguration.class);
+        if (Iterables.size(defaultConfigProvider) > 0) {
+            LOGGER.info("A default configuration is generated from an implementation.");
+            return Iterables.get(defaultConfigProvider, 0).get();
+        }
         if (defaultConfig == null) {
+            LOGGER.info("A default configuration is generated from a local file.");
```

Comment:
This is the expected behavior (runtime)? Consider changing the log level to debug